e2e/wdio/core/tests/table.e2e-spec.ts

```diff
@@ -168,12 +169,14 @@ describe('Table test suite', function() {
             const checkboxLength = getElementArrayLength(tableCheckboxesExample + markAllCheckboxes);
             for (let i = 0; i < checkboxLength; i++) {
                 scrollIntoView(tableCheckboxesExample + markAllCheckboxes, i);
-                click(tableCheckboxesExample + markAllCheckboxes, i);
+                browserIsFirefox() ? click(tableCheckboxesExample + markAllCheckboxesFF, i) :
+                    click(tableCheckboxesExample + markAllCheckboxes, i);
             }
 
             const checkboxLength_2 = getElementArrayLength(tableCheckboxesExample + tableRow);
             for (let i = 0; i < checkboxLength_2; i++) {
-                expect(getAttributeByName(tableCheckboxesExample + tableRow, 'aria-selected', i)).toBe('true');
+                expect(getAttributeByName(tableCheckboxesExample + tableRow, 'aria-selected', i))
+                    .toBe('true');
```

Comment:
add error message

Reply:
Well, this is not my implementation, just changed the formatting to not exceed max line length. But ok, I'll add it; next time let's try to add that comment on the PR implementing the code  ;)